examples/react-whiteboard/src/index.js

```diff
@@ -1,28 +1,12 @@
-import React from "react";
-import ReactDOM from "react-dom/client";
+import { createRoot } from "react-dom/client";
```

Comment:
Thx for upgrading this example to use modern React primitives, at the same time 👍